de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537690_sys_gh5856_Add_Payment_sysconfig.sql

```diff
@@ -0,0 +1,5 @@
+-- 2019-12-03T06:49:39.145Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,Created,CreatedBy,IsActive,ConfigurationLevel,AD_SysConfig_ID,Updated,UpdatedBy,Value,Name,AD_Org_ID,EntityType) VALUES (1000000,TO_TIMESTAMP('2019-12-03 08:49:38','YYYY-MM-DD HH24:MI:SS'),100,'Y','C',541292,TO_TIMESTAMP('2019-12-03 08:49:38','YYYY-MM-DD HH24:MI:SS'),100,'Y','de.metas.payment.autoAssignToOrderByExternalOrderId.enabled',0,'D')
```

Comment:
I thinks you'd better use configuration level `O` (organisation). That way, every AD_Org can overwrite this setting. 
(ofc it means that you need to provide the OrgId to `ISysConfigBL` when retrieving the sys-config, but that should be no problem)